src/test/java/seedu/address/logic/commands/tag/DeleteTagCommandTest.java

```diff
@@ -300,6 +300,11 @@ public ObservableList<Event> getFilteredEventList() {
         public ObservableList<Event> getUnfilteredEventList() {
             throw new AssertionError("This method should not be called.");
         }
+
+        @Override
+        public ObservableList<Task> getTaskDueSoonList() {
```

Comment:
test case should also be changed when we rename to `getTasksDueSoonList()`

Reply:
Yep, have changed accordingly.